src/main/webapp/app/entities/exercise/exercise.model.ts

```diff
@@ -1,7 +1,9 @@
-import { BaseEntity } from './../../shared';
+import { BaseEntity } from 'app/shared';
 import { Course } from '../course';
 import { Participation } from '../participation';
+import * as moment from 'moment';
 import { Moment } from 'moment';
+import { EntityArrayResponseType, EntityResponseType } from 'app/entities/exercise/exercise.service';
```

Comment:
I am not sure if it is a good idea that the model code depends on something defined in the service class. Therefore, I would suggest to move the static `convert...` functions into the exercise.service